A/argtable/argtable@3.2.2/build_tarballs.jl

```diff
@@ -0,0 +1,39 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+
+include("../common.jl")
+
+version = v"3.2.2"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource(
+        "https://github.com/argtable/argtable3.git",
+        "55c4f6285e2f9b2f0cc96ae8212b7b943547c3dd",
+    ),
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/argtable3
+install_license LICENSE
+CC=gcc
+CXX=g++
```

Comment:
Why?
```suggestion
```
This is also [the wrong way](https://docs.binarybuilder.org/stable/build_tips/#Using-GCC-on-macOS-and-FreeBSD) to set the compilers when using CMake

Reply:
Ah, apologies